docs/Image Uploader.md

```diff
@@ -4,7 +4,7 @@ You can drag and drop images to Chatterino or paste them from clipboard to uploa
 By default, the image uploader is turned off. You can enable by checking the `Chatterino Settings -> External Tools -> Image Uploader -> Enable image uploader` option and configuring per one of the below options.
 
 Note to advanced users: This module sends multipart-form requests via POST method, so uploading via SFTP/FTP won't work.
-However, popular hosts like [imgur.com](https://imgur.com) and [s-ul.eu](https://s-ul.eu) are supported. Scroll down to see example cofiguration.
+However, popular hosts like [imgur.com](https://imgur.com), [s-ul.eu](https://s-ul.eu) and [ctrlv.link](https://ctrlv.link) are supported. Scroll down to see example cofiguration.
```

Comment:
I'm not so sure about this edit. i.nuuls.com isn't included in this list, and it's a highly used website by a lot of users on Twitch.